use-timescale/integrations/apache-kafka.md

```diff
@@ -0,0 +1,247 @@
+   1. Send some events. For example, type the following:
+      ```bash
+      >Timescale Cloud
+      >How Cool
+      ```
+   2. In another terminal window, consume the events you just sent:
+      ```bash
+      bin/kafka-console-consumer.sh --topic accounts --from-beginning --bootstrap-server localhost:9092
+      ```
+      You see
+      ```bash
+      Timescale Cloud
+      How Cool
+     ```
+   
+</Procedure>
+
+Keep these terminals open, you use them to test the integration later.
+
+## Install the sink connector to communicate with $CLOUD_LONG
+
+To set up Kafka Connect server, plugins, drivers, and connectors:
+
+<Procedure>
+
+1. **Install the PostgreSQL connector**
+
+   In Terminal, in the root folder of your Kafka deployment, download and configure the PostgreSQL sink and driver.
+   ```bash
+   mkdir -p "plugins/camel-postgresql-sink-kafka-connector"
+   curl https://repo.maven.apache.org/maven2/org/apache/camel/kafkaconnector/camel-postgresql-sink-kafka-connector/3.21.0/camel-postgresql-sink-kafka-connector-3.21.0-package.tar.gz \
+   | tar -xzf - -C "plugins/camel-postgresql-sink-kafka-connector" --strip-components=1
+   curl  -H "Accept: application/zip" https://jdbc.postgresql.org/download/postgresql-42.7.5.jar -o  "plugins/camel-postgresql-sink-kafka-connector/postgresql-42.7.5.jar"
+   echo "plugin.path=`pwd`/plugins/camel-postgresql-sink-kafka-connector" >> "config/connect-distributed.properties"
+   echo "plugin.path=`pwd`/plugins/camel-postgresql-sink-kafka-connector" >> "config/connect-standalone.properties" 
+   ```
+   
+1. **Start Kafka Connect**
+
+    ```bash
+   export CLASSPATH=`pwd`/plugins/camel-postgresql-sink-kafka-connector/*
+   ./bin/connect-standalone.sh config/connect-standalone.properties
+   ```
+
+   Use the `-daemon` flag to run this process in the background.
+
+1. **Verify Kafka Connect is running**
+
+    In yet another another Terminal window, run the following command:  
+    ```bash
+    curl http://localhost:8083
+    ```
+    You see something like:
+    ```bash
+    {"version":"3.9.0","commit":"a60e31147e6b01ee","kafka_cluster_id":"J-iy4IGXTbmiALHwPZEZ-A"}
+    ```
+
+</Procedure>
+
+## Create a table in your $SERVICE_LONG to ingest Kafka events
+
+To prepare your $SERVICE_LONG for Kafka integration:
+
+<Procedure>
+
+1. ** [Connect][connect] to your $SERVICE_LONG **
+
+1. **Create a table to ingest Kafka events**
+
+   ```sql
+   CREATE TABLE accounts (created_at TIMESTAMPTZ DEFAULT NOW(),
+    name TEXT,
+    city TEXT);
+   ```
+
+1. **Turn the table into a hypertable**
+
+   ```sql
+   SELECT create_hypertable('accounts', 'created_at');
+   ```
+
+</Procedure>
+
+## Create the $CLOUD_LONG sink
+
+To create a $CLOUD_LONG sink in Apache Kafka: 
+
+<Procedure>
+
+
+1.  **Create the connection configuration**
+ 
+       1. Write the following configuration to `config/timescale-standalone-sink.properties`, then update the `<properties>` with your [connection details][connection-info].
```

Comment:
1. I think it should be made clear that you create this file in the `kafka root/config` directory.
2. If I create `timescale-standalone-sink` as a .properties file, I get an error at the next step saying it expects a json. If I create and pass a json with the same name, I get an error that Kafka Connect cannot find the specified connector class (`CamelPostgresqlsinkSinkConnector`) and that it detects only two connectors `org.apache.camel.kafkaconnector.CamelSinkConnector` and
`org.apache.camel.kafkaconnector.CamelSourceConnector`. AI suggests 1. checking whether the plugin is there (it is) 2. updating this file to refer to `org.apache.camel.kafkaconnector.CamelSinkConnector` instead, which I assume is not the goal here, as we install this Postgres connector specifically.

Reply:
Done